janusgraph-cql/pom.xml

```diff
@@ -291,19 +291,19 @@
                         <phase>test</phase>
                     </execution>
                     <execution>
-                        <id>${test.murmur}-serial-test</id>
+                        <id>cql-serial-test</id>
                         <goals>
                             <goal>test</goal>
                         </goals>
                         <phase>test</phase>
                         <configuration>
-                            <excludedGroups />
+                            <excludedGroups>none</excludedGroups>
```

Comment:
This is the place that causes trouble. If we go to any previous murmur-graphdb CI run, e.g. https://github.com/JanusGraph/janusgraph/runs/2312741486#step:6:1 we can verify that CQLEventualGraphTest, which is annotated as "SERIAL_TESTS", is not being run.

Applied the fix suggested by https://www.baeldung.com/maven-integration-test,

> Notice that we must set an exclude element to none to override the exclusion specified in the base configuration.

Using "none" (actually, any dummy value) works.

Reply:
Now we can see [murmur-graphdb](https://github.com/JanusGraph/janusgraph/pull/2569/checks?check_run_id=2316844127#step:6:283) and [scylladb-graphdb](https://github.com/JanusGraph/janusgraph/pull/2569/checks?check_run_id=2316844196#step:6:279) both run CQLEventualGraphTest.